packages/webapp/src/journey-validation/allocation/task-sections.js

```diff
@@ -58,11 +58,11 @@ const checkYourAnswers = {
 
 const taskSections = [
   taskSectionDefinition('Applicant information', [applicantDetails]),
+  taskSectionDefinition('Development project information', [addDevlopmentProjectInformation]),
```

Comment:
This section should be in the 'Development information' section below. So remove this line and add `addDevlopmentProjectInformation` to the below section